src/sentry/static/sentry/app/components/forms/passwordField.tsx

```diff
@@ -54,13 +65,17 @@ export default class PasswordField extends InputField {
     );
   };
 
-  startEdit = e => {
+  startEdit = (e: React.MouseEvent<HTMLAnchorElement>) => {
     e.preventDefault();
     this.setState({
       editing: true,
     });
   };
 
+  get prefix() {
+    return this.props.prefix || PasswordField.defaultProps.prefix;
```

Comment:
I'm not a fan of this solution because we're sort of re-implementing `defaultProps`, is there something better we can do? /cc @dashed @leedongwei @EvanPurkhiser ?

I'm also in favor of not having this particular issue block the PR fwiw.

Reply:
Fixed this by making the prop required